contracts/SportMarkets/Parlay/ParlayMarketData.sol

```diff
@@ -14,6 +14,9 @@ import "./ParlayMarket.sol";
 contract ParlayMarketData is Initializable, ProxyOwned, ProxyPausable {
     using AddressSetLib for AddressSetLib.AddressSet;
     AddressSetLib.AddressSet internal _knownMarkets;
+
+    uint private constant ONE = 1e18;
```

Comment:
we need to create some kind of Constant contract which will hold this all constant variables also i would like to have one place with all contracts like `ContractManager.sol` which will store all contract addresses and then all contract will extend/import that contract.

But that all is for future.

Reply:
Agreed